api/src/main/java/keywhiz/api/model/SecretSeries.java

```diff
@@ -34,18 +36,18 @@
  */
 @AutoValue
 public abstract class SecretSeries {
-  public static SecretSeries of(
-      long id,
-      String name,
-      String owner,
-      @Nullable String description,
-      ApiDate createdAt,
-      @Nullable String createdBy,
-      ApiDate updatedAt,
-      @Nullable String updatedBy,
-      @Nullable String type,
-      @Nullable Map<String, String> generationOptions,
-      @Nullable Long currentVersion) {
+  @JsonCreator public static SecretSeries of(
+      @JsonProperty("id") long id,
+      @JsonProperty("name") String name,
```

Comment:
Unlike many other API classes, this class was apparently never directly returned as a result so it had none of the Jackson annotations required for serialization. Now that we have a new API which returns deleted secret series (if only to support testing), I've added annotations and tests to verify that (de)serialization works as intended.